### PR TITLE
Simplify boolean expr

### PR DESCRIPTION
**Pull request by dennisklein**
for the fun of it

### CODE REVIEW

**Review by ChristianTackeGSI — APPROVED** [review]
Please add `readability-simplify-boolean-expr` to `.clang-tidy`, so that we will not miss anything again.

File: generators/FairYPtGenerator.cxx [hunk 1]
```diff
@@ -46,8 +46,8 @@ void FairYPtGenerator::SetYPt(const TH2D &yPt)
 
 Bool_t FairYPtGenerator::Init()
 {
-    if (FairBaseMCGenerator::Init() == kFALSE)
-        return kFALSE;
+    if (!FairBaseMCGenerator::Init())
+        return fFALSE;
```
Comment (ChristianTackeGSI) [inline generators/FairYPtGenerator.cxx:50]: Typo…

I wonder, why this compiles on some of our machines.

```suggestion
        return false;
```

Comment (dennisklein) [reply generators/FairYPtGenerator.cxx]: > I wonder, why this compiles on some of our machines.

I suspect overlapping success signals from the previous CI builds.

Comment (dennisklein) [reply generators/FairYPtGenerator.cxx]: Ya, that's it, https://alfa-ci.gsi.de/blue/organizations/jenkins/FairRootGroup%2FFairRoot/detail/PR-1267/3/pipeline/ here you see that all really failed.

**Review by ChristianTackeGSI — APPROVED** [review]

**Pull request merged**